returnn/datasets/hdf.py

```diff
@@ -355,6 +377,33 @@ def _get_tag_by_real_idx(self, real_seq_idx):
     s = self._decode(s)
     return s
 
+  def _get_real_idx_by_tag(self, seq_tag):
```

Comment:
I don't think such a helper function is needed. Just call `_update_tag_idx` and access `_tag_idx` directly.